fbw-common/src/systems/instruments/src/OANC/Oanc.tsx

```diff
@@ -911,8 +911,13 @@ export class Oanc<T extends number> extends DisplayComponent<OancProps<T>> {
 
   private lastTime = 0;
 
-  private projectCoordinates(coordinates: Coordinates): [number, number] {
-    return globalToAirportCoordinates(this.arpCoordinates, coordinates);
+  /**
+   *
+   * @param projectedCoordinates Output argument: Write projected coordinates here
+   * @param coordinates coordinates to be transformed
+   */
+  private projectCoordinates(projectedCoordinates: Position, coordinates: Coordinates) {
```

Comment:
Better to name it out to make it clear that it's the out param, and it's also nice to return it so you can chain calls, or put it in a handy local scoped name if you're using a cache object. Also nice to put the out param at the end by convention.

Reply:
done